src/solid-shapes/include/iDynTree/ModelTransformersSolidShapes.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * SPDX-FileCopyrightText: 2021 Istituto Italiano di Tecnologia (IIT)
+ * SPDX-License-Identifier: LGPL-2.1-or-later
+ */
+
+
+/**
+ * \file ModelTransformersSolidShapes.h
+ *  \brief Collection of function to modify model, related to solid shapes.
+ *
+*/
+
+
+#ifndef IDYNTREE_MODEL_TRANSFORMERS_SOLID_SHAPES_H
+#define IDYNTREE_MODEL_TRANSFORMERS_SOLID_SHAPES_H
+
+namespace iDynTree
+{
+class Model;
+class SensorsList;
+
+enum ApproximateSolidShapesWithPrimitiveShapeConversionType {
+    ConvertSolidShapesWithEnclosingAxiAlignedBoundingBox
```

Comment:
Is an ``s`` missing (``ConvertSolidShapesWithEnclosingAxisAlignedBoundingBox``)?

Reply:
Like `ConvertSolidShapesWithEnclosingAxisAlignedBoundingBoxes` ?

Reply:
Yes exactly, between the parenthesis I wrote the suggestion, but indeed the ``s`` was difficult to see :grin:

Reply:
I may be missing something but between the parenthesis there is no s.

Reply:
Ah, now I see, where you inserted the other ``s``. I meant ``Axi`` -> ``Axis``.

Reply:
Ahhhhh. Ok!

Reply:
The funny thing is that in https://github.com/robotology/idyntree/pull/941#discussion_r749103126 you did add the ``s`` in ``Axis``, but then I completely missed the ``es`` in ``Boxes``, so I was super confused :joy:

Reply:
Fixed in https://github.com/robotology/idyntree/pull/941/commits/73b387927fbca937bef295be7ab6858ac960f45f .